ensembl/conf/json/macaca_fascicularis_vcf.json

```
@@ -0,0 +1,17 @@
+{
+  "collections": [
+    {
+      "id": "crab_macaque_EVA",
+      "source_name": "EVA",
+      "description": "Variants from EVA",
+      "species": "macaca_fascicularis",
+      "assembly": "Macaca_fascicularis_6.0",
+      "type": "remote",
+      "use_as_source" : 1,
+      "filename_template" : "http://ftp.ebi.ac.uk/pub/databases/eva/rs_releases/release_3/by_species/macaca_fascicularis/GCA_011100615.1/GCA_011100615.1_current_ids.vcf.gz",
+      "chromosomes": [
+        "1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","X"
+      ]
+    }
```

Include GERP conservation scores:

```suggestion
    },
    {
      "id": "91_mammals.gerp_conservation_score",
      "species": "macaca_fascicularis",
      "assembly": "Macaca_fascicularis_6.0",
      "type": "remote",
      "filename_template": "/91_mammals.gerp_conservation_score/gerp_conservation_scores.macaca_fascicularis.Macaca_fascicularis_6.0.bw",
      "annotation_type": "gerp"
    }
```